antlr4-wrapper.xml

```diff
@@ -12,9 +12,17 @@
 
         See AntlrGeneratedParserBase
 
+        It also uses the following maven properties:
+         - ant.contrib.jar:  Location of the ant-contrib jar
 -->
+    <taskdef resource="net/sf/antcontrib/antcontrib.properties">
+        <classpath>
+            <pathelement location="${ant.contrib.jar}"/>
+        </classpath>
```

Comment:
We don't need the extra classpath config, as we put ant-contrib on the antrun-plugin's classpath in pom.xml. So, we can make this a lot easier - ant-contrib is always available. No need for the property `ant.contrib.jar` to be set by each module.

Reply:
Thanks for the insight